css/bootstrap/darkstrap-0.9.3.css

```diff
@@ -4786,7 +4786,8 @@ tbody.collapse.in {
 
 .navbar-brand {
   float: left;
-  padding: 15px 15px;
+  padding: 3px;
+  padding-left: 8px;
```

Comment:
This is the generated CSS from the Darkstrap project. Please move the custom CSS into css/bootstrap/privatebin.css for all Bootstrap/Darkstrap themes and check what needs to be done in css/privatebin.css for the page theme to integrate the logo there too.